webapp/tests/karma/ts/services/db-sync.service.spec.ts

```diff
@@ -37,13 +40,15 @@ describe('DBSync service', () => {
   beforeEach(() => {
     clock = sinon.useFakeTimers();
 
-    replicationResult = () => Promise.resolve();
+    replicationResultTo = Promise.resolve();
```

Comment:
I remember when doing the Angular migration to suffer with a change like this, for some reason that I don't remember `someResult = Promise.resolve();` didn't work well, and changing it to `someResult = () => Promise.resolve();` solved the problem, but you are doing the opposite here, I guess you know what you are doing :raised_eyebrow:

Reply:
;) 
If I had screwed something up, tests would start failing, wouldn't they?

Reply:
Yea , what I remember is that actually the tests were failing silently, there were unhandled promise rejections in the console, but sill the tests were running OK because the values checked were wrong, and trying to set the right value I received errors that in the end when I saw those unhandled errors in the log I realized what was the problem, but not sure the exact scenario now.

To be completely sure is not happening here I gave it a try running the tests locally and checking the logs, and I didn't see any unhandled rejection so we are fine :smile: